server/py/services/api/utils/endpoints.py

```diff
@@ -34,6 +34,7 @@ async def start_model_endpoint_creation_background_task(
     returned_background_tasks = mlrun.common.schemas.BackgroundTaskList(
         background_tasks=[]
     )
+    model_endpoints_instructions = []
```

Comment:
I dont think we really need this

Reply:
We need it to pass to `_create_model_endpoint_background_task`, and to extract and return `model_endpoint_uids`.

Reply:
but we create those inside `_create_model_endpoints_instructions` and it comes before `_create_model_endpoint_background_task`

Reply:
We need the empty list in case we didn't go into the conditional block.